module/actor/sheets/base.js

```diff
@@ -411,22 +405,23 @@ export default class ActorSheetSS2e extends ActorSheet {
   /* -------------------------------------------- */
 
   /**
-   * Handle rolling of an item from the Actor sheet, obtaining the Item instance and dispatching to it's roll method
+   * Handle toggling and items expanded description.
+   * @param {Event} event   Triggering event.
    * @private
    */
   async _onItemSummary(event) {
     event.preventDefault();
-    const li = event.currentTarget.closest(".item");
-    const item = this.actor.items.get(li.dataset.itemId);
-    const chatData = item.getChatData({ secrets: this.actor.owner });
+    const li = $(event.currentTarget).parents(".item");
+    const item = this.actor.items.get(li.data("item-id"));
+    const chatData = item.getChatData({secrets: this.actor.isOwner});
 
     // Toggle summary
     if (li.hasClass('expanded')) {
-      const summary = li.children('.item-summary');
+      let summary = li.children('.item-summary');
       summary.slideUp(200, () => summary.remove());
     } else {
-      const div = $(`<div class="item-summary">${chatData.description}</div>`);
-      const metadata = $(
+      let div = $(`<div class="item-summary">${chatData.description}</div>`);
+      let metadata = $(
```

Comment:
Since the variables are not intended to be redeclared later in the code, `const` is the correct variable scope.